File: core/src/com/broll/gainea/server/core/goals/impl/e2/G_AllIslands.java
package com.broll.gainea.server.core.goals.impl.e2;

import com.broll.gainea.server.core.goals.OccupyGoal;
import com.broll.gainea.server.core.goals.GoalDifficulty;
import com.broll.gainea.server.core.map.impl.IcelandMap;

public class G_AllIslands extends OccupyGoal {
    public G_AllIslands() {
        super(GoalDifficulty.MEDIUM, "Erobere alle Inseln");
    }

    @Override
    protected void initOccupations() {
        occupy(IcelandMap.Islands.values());
    }

}

File: server/src/main/com/broll/gainea/server/core/goals/impl/e2/G_AllIslands.kt
package com.broll.gainea.server.core.goals.impl.e2

import com.broll.gainea.server.core.goals.GoalDifficulty
import com.broll.gainea.server.core.goals.OccupyGoal
import com.broll.gainea.server.core.map.impl.IcelandMap

class G_AllIslands : OccupyGoal(GoalDifficulty.MEDIUM, "Erobere alle Inseln") {
    override fun initOccupations() {
        occupy(*IcelandMap.Islands.entries.toTypedArray())
    }
}
